shared/common-adapters/markdown/service-decoration.tsx

```diff
@@ -11,6 +14,29 @@ import MaybeMention from '../../chat/conversation/maybe-mention'
 import Text, {StylesTextCrossPlatform} from '../text'
 import {StyleOverride} from '.'
 
+type KeybaseLinkProps = {
+  link: string
+  styleOverride: StyleOverride
+  styles: {[K in string]: StylesTextCrossPlatform}
+}
+
+const KeybaseLink = (props: KeybaseLinkProps) => {
+  const dispatch = Container.useDispatch()
+  const onClick = React.useCallback(() => dispatch(DeeplinksGen.createLink({link: props.link})), [dispatch])
+
+  return (
+    <Text
+      className="hover-underline"
+      type="BodyPrimaryLink"
+      style={Styles.collapseStyles([props.styles.wrapStyle, linkStyle, props.styleOverride.link])}
```

Comment:
Since we're only reading `wrapStyle` from `styles`, would it make more sense to just accept `wrapStyle` as a prop? Or would it make sense to define the `StyleOverride` values in `styles`? It seems a little weird to have two props that do the same thing here.

Reply:
@nathunsmitty Thanks, perhaps like this? b9a70bb1a9